caterva2/tests/test_api.py

```diff
@@ -386,7 +329,7 @@ def test_append(auth_client, fields, fill_auth, examples_dir):
     "slice_",
     [1, slice(None, 1), slice(0, 10), slice(10, 20), slice(None), slice(10, 20, 1)],
 )
-def test_dataset_getitem_fetch(slice_, examples_dir, client):
+def test_dataset_getitem_fetch(slice_, examples_dir, client, fill_public):
```

Comment:
`fill_public` does not seem to be used here.

Reply:
The return value of fill_public is not used, but this fixture copies data (to the @public root) that is required for the test to pass.